src/database/schema/ChannelListSchema.ts

```diff
@@ -398,6 +398,16 @@ class ChannelList implements BaseDbSchema {
     }
   };
 
+  static async updateChannelList(db: SQLiteDatabase, channelId: string, channelType: string) {
+    const query = `UPDATE channel_lists SET channel_type="${channelType}" WHERE id="${channelId}"`;
+    await db.executeSql(query);
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The SQL query in `updateChannelList` is vulnerable to SQL injection attacks because it directly concatenates user-provided strings into the query. It's better to use parameterized queries to prevent this security issue.
```diff
-    const query = `UPDATE channel_lists SET channel_type="${channelType}" WHERE id="${channelId}"`;
-    await db.executeSql(query);
+    const query = 'UPDATE channel_lists SET channel_type=? WHERE id=?';
+    await db.executeSql(query, [channelType, channelId]);
```



<!-- This is an auto-generated comment by OSS CodeRabbit -->